test/sync/src/androidTest/kotlin/io/realm/kotlin/test/mongodb/shared/internal/KtorNetworkTransportTest.kt

```diff
@@ -30,6 +30,7 @@ import kotlin.test.assertEquals
 
 private const val BASE_URL = "http://127.0.0.1:8888" // URL to command server
 
+@Ignore // FIXME we could test it with a local ktor server
```

Comment:
Ignoring this test is a bit problematic as it revealed some rather big problems with Ktor in the past.

Reply:
Yes, I wanted to enable them with some solution, I had in mind to use a local ktor server or use atlas static pages.